packages/itwin/tree-widget/src/components/TreeSelector.tsx

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import "./TreeSelector.scss";
+import type { SelectOption } from "@itwin/itwinui-react";
+import { useMemo, useState } from "react";
+import { MenuItem, Select } from "@itwin/itwinui-react";
+import type { TreeRenderProps } from "./SelectableTree";
+
+export interface TreeContentDefinition {
+  id: string;
+  label: string;
+  render: (props: TreeRenderProps) => React.ReactNode;
+}
+
+export interface TreeSelectorProps {
+  defaultSelectedContentId: string;
+  trees: TreeContentDefinition[];
+  selectAriaLabel?: string;
+  disabled?: boolean;
```

Comment:
These are not used in `SelectableTree`. Maybe remove them?

Reply:
Removed.